### DISCUSSION TITLE
Use LrcLib API to fetch lyrics if lyrics are not tagged in the file.

### DISCUSSION THREAD

**Discussion by dweymouth** [navidrome/navidrome#2892] [Ideas]
Once external .lrc files are supported, they should be treated as a preferred source as well. Ie LrcLib should only be queried if neither embedded nor .lrc lyrics are available.

**Comment by felixconsulting** [comment]
The android client [Chora](https://github.com/CraftWorksMC/Chora) has this feature already, but it is only client side. I agree this would be a great feature to have.
